libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.java

```diff
@@ -181,8 +182,16 @@ private void restartIfUserSwitched() {
      */
     private void onResumeNotLoggedIn() {
 
+        BootConfigValidationResult validationResult = BootConfig.validateBootConfig(bootconfig);
```

Comment:
I coded this to ultimately allow validation of any boot config.  But since the only thing we're validating at the moment is hybrid configuration, I didn't plug this validation step in anywhere else.